gradle.properties

```diff
@@ -18,11 +18,11 @@ containerTests=false
 # - CodeCache normally defaults to a very small size. Increasing it from platform defaults of 32-48m
 #   because of how many classes can be loaded into memory and then cached as native compiled code
 #   for a small speed boost.
-org.gradle.jvmargs=-Dfile.encoding=UTF-8 -XX:+UseG1GC -XX:SoftRefLRUPolicyMSPerMB=1 -XX:ReservedCodeCacheSize=256m -XX:+HeapDumpOnOutOfMemoryError -Xmx1g -Xms1g
+org.gradle.jvmargs=-Dfile.encoding=UTF-8 -XX:+UseG1GC -XX:SoftRefLRUPolicyMSPerMB=1 -XX:ReservedCodeCacheSize=256m -XX:+HeapDumpOnOutOfMemoryError -Xmx2g -Xms2g
```

Comment:
Any idea what is motivating this?

Reply:
Sorry, it's meant to be 4gb. From the tuned project I cribbed from.

But I was getting failures I thought were memory related. And was testing.

I still don't think we can have any 4x4gb

Reply:
reverted from the PR